joomla/joomla-cms
# Change param show_search_form of com_finder

This commit adds the possibility to edit the `show_search_form` parameter of the com_finder component.

### Summary of Changes
That parameter is actually being already in current (and past) versions, see https://github.com/joomla/joomla-cms/blob/17c3ec7b9ded2cf7bf3e6b5295e33d94ae4043f8/components/com_finder/views/search/tmpl/default.php#L28 - but there was no possibility present to set this parameter to a value.
Usage of that parameter was introduced way back with baa509338c8b92e41999fbc8d79ac755f9b6a761 but already there you could not set this parameter to a value. This commit fixes this.

### Testing Instructions
Apply the patch, set the parameter to a value (either 'Show' or 'Hide'), either in the global component configuration, or when creating/editing a menulink to the com_finder component. Then check whether the search form is shown or not.

### Expected result
Depending on the parameter, the search form will be shown or not.

### Documentation Changes Required
No.
